### DISCUSSION TITLE
io.elementary.Platform Flatpak runtime and io.elementary.Sdk Flatpak runtime fail to update

### DISCUSSION THREAD

**Discussion by jhpak725** [elementary/flatpak-platform#40] [Q&A]
whether trying to update via AppCenter or command line, the update fails and I get a new notification saying an update is available. I have included the verbose log while updating, can anyone see what's wrong?

jhpak@kamek:~/Downloads$ flatpak --verbose --ostree-verbose update
F: No installations directory in /etc/flatpak/installations.d. Skipping
F: Opening system flatpak installation at path /var/lib/flatpak
F: Opening user flatpak installation at path /home/jhpak/.local/share/flatpak
Looking for updates…
F: Fetching summary index file for remote ‘appcenter’
F: Loading https://flatpak.elementary.io/repo/summary.idx using libsoup
F: Received 902 bytes
F: Loaded indexed summary file 076227bec31af436a4dc15eee49269ed9c6685c34d2c315e8ba32ff04453f87a from cache for remote ‘appcenter’
F: Transaction: update appcenter:app/io.elementary.calculator/x86_64/master[*]
F: Transaction: update appcenter:app/org.gnome.Epiphany/x86_64/master[*]
F: Transaction: update appcenter:app/org.gnome.Evince/x86_64/master[*]
F: Transaction: update appcenter:runtime/io.elementary.Platform.Locale/x86_64/daily[*]
F: Transaction: update appcenter:runtime/io.elementary.Platform/x86_64/daily[*]
F: Transaction: update appcenter:runtime/io.elementary.Sdk.Locale/x86_64/daily[*]
F: Transaction: update appcenter:runtime/io.elementary.Sdk/x86_64/daily[*]
F: Transaction: update appcenter:runtime/io.elementary.calculator.Locale/x86_64/master[*]
F: Transaction: update appcenter:runtime/org.gnome.Epiphany.Locale/x86_64/master[*]
F: Transaction: update appcenter:runtime/org.gnome.Evince.Locale/x86_64/master[*]
F: Fetching summary index file for remote ‘flathub’
F: Loading https://dl.flathub.org/repo/summary.idx using libsoup
F: Received 2737 bytes
F: Loaded indexed summary file 86863b6adc0c7b083b736976dd95197cfd3acf17f71d695979622d70fecdb909 from cache for remote ‘flathub’
F: Transaction: update flathub:app/io.mpv.Mpv/x86_64/stable[*]
F: Transaction: update flathub:app/org.gnome.gedit/x86_64/stable[*]
F: Transaction: update flathub:app/org.libreoffice.LibreOffice/x86_64/stable[*]
F: Transaction: update flathub:app/org.videolan.VLC/x86_64/stable[*]
F: Transaction: update flathub:runtime/io.mpv.Mpv.Locale/x86_64/stable[/en]
F: Transaction: update flathub:runtime/org.freedesktop.Platform.GL.default/x86_64/20.08[*]
F: Transaction: update flathub:runtime/org.freedesktop.Platform.Locale/x86_64/20.08[/en]
F: Transaction: update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/20.08[*]
F: Transaction: update flathub:runtime/org.freedesktop.Platform.openh264/x86_64/2.0[*]
F: Transaction: update flathub:runtime/org.freedesktop.Platform/x86_64/20.08[*]
F: Transaction: update flathub:runtime/org.gnome.Platform.Locale/x86_64/40[/en]
F: Transaction: update flathub:runtime/org.gnome.Platform/x86_64/40[*]
F: Transaction: update flathub:runtime/org.gnome.gedit.Locale/x86_64/stable[/en]
F: Transaction: update flathub:runtime/org.kde.Platform.Locale/x86_64/5.15[/en]
F: Transaction: update flathub:runtime/org.kde.Platform/x86_64/5.15[*]
F: Transaction: update flathub:runtime/org.libreoffice.LibreOffice.Locale/x86_64/stable[/en]
F: Transaction: update flathub:runtime/org.videolan.VLC.Locale/x86_64/stable[/en]
F: Looking for remote metadata updates for appcenter
F: marking op update:runtime/org.gnome.Evince.Locale/x86_64/master resolved to 138190b7c685a3a3d1dc3cd2b13e98e56d347d6b5e941f7642aef1fd14095beb
F: marking op update:runtime/org.gnome.Epiphany.Locale/x86_64/master resolved to 40627df9239fd494fbf84c28330ac088bbabbb055fac9bfb2fbe50fbfc314c92
F: marking op update:runtime/io.elementary.calculator.Locale/x86_64/master resolved to c56abebd719387c820b4026049b0a825a73d756ef8e8e86c2f1e61a1382f265e
F: marking op update:runtime/io.elementary.Sdk/x86_64/daily resolved to d608cc55bec765e3b118009acfa5e48326227cc142da6fb9619461918b72bb41
F: marking op update:runtime/io.elementary.Sdk.Locale/x86_64/daily resolved to b0e333af822aad4ff51a702403c943d1f1052acb8dcb868e4b6f700520ae46b3
F: marking op update:runtime/io.elementary.Platform/x86_64/daily resolved to fbfbbc434dd700f3ba976fc180013d04318cac46b33f48eb69f000f2f60d2ef3
F: marking op update:runtime/io.elementary.Platform.Locale/x86_64/daily resolved to e32f5ddb310639663e9c156bd5423ab5f96dd5858db5ae28420e23900dd0e3c4
F: marking op update:app/org.gnome.Evince/x86_64/master resolved to 022809bbd47e2966a2d65bd95d0caeeba212f1e43dd95363789d2737a8c7ada8
F: marking op update:app/org.gnome.Epiphany/x86_64/master resolved to cbfe7ce313f8ccb3ad086f0852449b69767d389da44fc465c2b6a0b31f4e22f1
F: marking op update:app/io.elementary.calculator/x86_64/master resolved to 3276a5b2f361d850675206343fb8e1cdc96888249ae7027bacdf8cb6afa321e5
F: Transaction: install/update appcenter:runtime/org.freedesktop.Platform.GL.default/x86_64/20.08[$old]
F: Transaction: install/update appcenter:runtime/io.elementary.Sdk.Locale/x86_64/daily[*]
F: Transaction: install/update appcenter:runtime/org.freedesktop.Platform.GL.default/x86_64/20.08[$old]
F: Transaction: install/update appcenter:runtime/io.elementary.Platform.Locale/x86_64/daily[*]
F: Transaction: install/update appcenter:runtime/org.gnome.Evince.Locale/x86_64/master[*]
F: Transaction: install/update appcenter:runtime/org.gnome.Epiphany.Locale/x86_64/master[*]
F: Transaction: install/update appcenter:runtime/io.elementary.calculator.Locale/x86_64/master[*]
F: marking op install/update:runtime/org.freedesktop.Platform.GL.default/x86_64/20.08 resolved to d7708cd178ce95fbd5ca903f767eeaac82b82713ca11e54421f2cf0a228b03ac

 1.	   	org.freedesktop.Platform.GL.default	20.08	i	appcenter	< 489.3 MB

Proceed with these changes to the system installation? [Y/n]: y
Installing…                                                                                                                    F: Calling system helper: GetRevokefsFd
F: Calling system helper: GetRevokefsFd
F: flatpak_dir_pull: Using commit d7708cd178ce95fbd5ca903f767eeaac82b82713ca11e54421f2cf0a228b03ac for pull of ref runtime/org.freedesktop.Platform.GL.default/x86_64/20.08 from remote appcenter
F: Preparing transaction in repository 0x557771026d10
F: Reusing tmpdir staging-543c30e5-a746-45ea-a172-a353a1558a1c-JZn1Qa
F: starting fetch of delta index delta-indexes/13/CM0XjOlfvVypA_dn7qrIK4JxPKEeVEIfLPCiKLA6w.index

Installing…                        0%  0 bytes/s
Installing…                        0%F: _ostree_fetcher_should_retry_request: error: unset, n_retries_remaining: 5
F: starting fetch of d7708cd178ce95fbd5ca903f767eeaac82b82713ca11e54421f2cf0a228b03ac.commit (detached)

Installing… ▍                      2%  0 bytes/sF: fetch of d7708cd178ce95fbd5ca903f767eeaac82b82713ca11e54421f2cf0a228b03ac.commit (detached) complete
F: _ostree_fetcher_should_retry_request: error: unset, n_retries_remaining: 5
F: Adding GPG keyring data from appcenter.trustedkeys.gpg to verifier
F: GnuPG >= 2.1.17, skipping gpg-agent cleanup in /tmp/ostree-gpg-gkpPnz
F: pull: idle, exiting mainloop

Installing… ████████████████████ 100%  336 bytes/sF: Calling system helper: CancelPull
F: Calling system helper: CancelPull
Warning: Commit metadata for runtime/org.freedesktop.Platform.GL.default/x86_64/20.08 not matching expected metadata             
Installation complete.                                                                                                           
F: Looking for remote metadata updates for flathub
F: marking op update:runtime/org.videolan.VLC.Locale/x86_64/stable resolved to 4f8607e50869001b9847eb0885e483dfba8e24cc5aafd1a364ebe81ba2503917
F: marking op update:runtime/org.libreoffice.LibreOffice.Locale/x86_64/stable resolved to 7b954670eaf7686650ceb27d2d776e0656c3429d54b2823e1e1a8bfad2369a03
F: marking op update:runtime/org.kde.Platform/x86_64/5.15 resolved to d8315965a89d63491339aa3fa827460a208d4cb702e01de32416d03faced960c
F: marking op update:runtime/org.kde.Platform.Locale/x86_64/5.15 resolved to f8cad41da8048b4001dfade78b4c234e6b3bccfd7db714310dbaae0b31077561
F: marking op update:runtime/org.gnome.gedit.Locale/x86_64/stable resolved to d8ff8ec33733a8f520ad787d2fba565ef38c497cb5f7923bd7e42db0859e6a5a
F: marking op update:runtime/org.gnome.Platform/x86_64/40 resolved to 3eaf98a3f031def1503dbf8c62e6d8741c6321422c6f0a39800e2a043f2108fc
F: marking op update:runtime/org.gnome.Platform.Locale/x86_64/40 resolved to 6c06ec42e57493d9065aee1a389de1cadee587b193a1c55260c8309a295e558c
F: marking op update:runtime/org.freedesktop.Platform/x86_64/20.08 resolved to a4f37bb933cf4792d472db2055886427d55074cdaf5bd8e4397dc3e2bc0305e1
F: marking op update:runtime/org.freedesktop.Platform.openh264/x86_64/2.0 resolved to 73f998362a6fc0d57e0c7e83e928d32b0ec14d10d0d94291033976bdcecc6b6b
F: marking op update:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/20.08 resolved to 118043f4ad0d7a1fd1e9b4e0a02146dd2ba1afd7fe50e0b3b02502c4cf2f5fe8
F: marking op update:runtime/org.freedesktop.Platform.Locale/x86_64/20.08 resolved to 6561b7ada9a8a4f9c33ff347bdf9424668b77a08af3cda4548aa77b77e341256
F: marking op update:runtime/org.freedesktop.Platform.GL.default/x86_64/20.08 resolved to a35dffb561996166d74b33de9768d6dcf34c62e8c728cbb1aa42f9f80cad1f5a
F: marking op update:runtime/io.mpv.Mpv.Locale/x86_64/stable resolved to cfeca107941fd2a3650051eed7492fb9026fb5176ed5ac8df3e78bf742e3694b
F: marking op update:app/org.videolan.VLC/x86_64/stable resolved to 5d5020575acac3d712b3bb20b69cde479a918d7778a3d055ead464e4fc0578a0
F: marking op update:app/org.libreoffice.LibreOffice/x86_64/stable resolved to e88bae337db226b2a1269eeb92715116be1f99d97d8fbfe717e55951019549f8
F: marking op update:app/org.gnome.gedit/x86_64/stable resolved to cd0da06d4896fabaa97f2fdecae39b5a5efe8461b0784f2ff2b17610994120af
F: marking op update:app/io.mpv.Mpv/x86_64/stable resolved to a1f6e28e1b6c9e95aaf4fbaa53a3a9a4caa278e30f200148cec3fb2977614cfe
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.GL.default/x86_64/20.08[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/20.08[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.openh264/x86_64/2.0[*]
F: Transaction: install/update flathub:runtime/org.kde.Platform.Locale/x86_64/5.15[/en]
F: Transaction: install/update flathub:runtime/org.gnome.Platform.Locale/x86_64/40[/en]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.GL.default/x86_64/20.08[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/20.08[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.openh264/x86_64/2.0[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.GL.default/x86_64/20.08[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.Locale/x86_64/20.08[/en]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/20.08[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.openh264/x86_64/2.0[*]
F: Transaction: install/update flathub:runtime/org.videolan.VLC.Locale/x86_64/stable[/en]
F: Transaction: install/update flathub:runtime/org.libreoffice.LibreOffice.Locale/x86_64/stable[/en]
F: Transaction: install/update flathub:runtime/org.gnome.gedit.Locale/x86_64/stable[/en]
F: Transaction: install/update flathub:runtime/io.mpv.Mpv.Locale/x86_64/stable[/en]
F: appcenter:x86_64 appstream age 2494 is less than ttl 86400
F: flathub:x86_64 appstream age 2487 is less than ttl 86400
F: flathub:x86_64 appstream age 2498 is less than ttl 86400

**Comment by cassidyjames** [comment]
This was a temporary issue as we worked out the server infrastructure. It should be resolved now.

**Comment by ryonakano** [comment]
Closing as it seems no longer reproducible.
